test/download-test.js

```diff
@@ -273,6 +274,28 @@ test('error during download should fail with no dangling temp file', function (t
   })
 })
 
+test('should fail if abi is system abi with invalid binary', function (t) {
+  var opts = getOpts()
+  opts.abi = rc.abi
```

Comment:
Here's the link to `rc.abi` which should cause a failing test, since it's set in the next commit.